ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -634,4 +704,13 @@ func verifyContainerConfig(t *testing.T, expected, actual adapter.ContainerConfi
 	assert.ElementsMatch(t, expected.Ulimits, actual.Ulimits, "Expected Ulimits to match")
 	assert.Equal(t, expected.User, actual.User, "Expected User to match")
 	assert.Equal(t, expected.WorkingDirectory, actual.WorkingDirectory, "Expected WorkingDirectory to match")
+	if expected.HealthCheck != nil && actual.HealthCheck != nil {
+		assert.ElementsMatch(t, aws.StringValueSlice(expected.HealthCheck.Command), aws.StringValueSlice(actual.HealthCheck.Command), "Expected healthcheck command to match")
+		assert.Equal(t, aws.Int64Value(expected.HealthCheck.Interval), aws.Int64Value(actual.HealthCheck.Interval), "Expected healthcheck interval to match")
+		assert.Equal(t, aws.Int64Value(expected.HealthCheck.Retries), aws.Int64Value(actual.HealthCheck.Retries), "Expected healthcheck retries to match")
+		assert.Equal(t, aws.Int64Value(expected.HealthCheck.StartPeriod), aws.Int64Value(actual.HealthCheck.StartPeriod), "Expected healthcheck start_period  to match")
+		assert.Equal(t, aws.Int64Value(expected.HealthCheck.Timeout), aws.Int64Value(actual.HealthCheck.Timeout), "Expected healthcheck timeout to match")
```

Comment:
Based on previous discussion of `assert.Equal`, `aws.Int64Value` is probably unnecessary here.